nautobot/virtualization/filters.py

```diff
@@ -90,11 +123,11 @@ class ClusterFilterSet(NautobotFilterSet, TenancyFilterSet):
         to_field_name="slug",
         label="Cluster type (slug)",
     )
-    tag = TagFilter()
+    tags = TagFilter()
```

Comment:
@gsnider2195 @jathanism

We aren't doing filter renames at this point, correct? But are we providing the corrected name as a part of this housekeeping?

Reply:
It's my understanding that we are not make changes to existing filters, just adding missing filters. I'll provide @timizuoebideri1 with the test that I used to identify all the missing filters.